packages/flex-plugin-utils-http/src/__tests__/http.test.ts

```diff
@@ -60,37 +70,45 @@ describe('HttpClient', () => {
   });
 
   describe('getUserAgent', () => {
-    // eslint-disable-next-line  global-require, @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
-    const pkg = require('../../package.json');
+    beforeEach(() => {
+      jest.spyOn(fsScripts, 'getDependencyVersion').mockReturnThis();
+    });
 
     it('should return default user-agent for node if nothing is set', () => {
       const isNode = jest.spyOn(env, 'isNode').mockReturnValue(true);
+      const isCI = jest.spyOn(env, 'isCI').mockReturnValue(false);
 
       // @ts-ignore
       const userAgent = HttpClient.getUserAgent({});
 
       expect(isNode).toHaveBeenCalledTimes(1);
+      expect(isCI).toHaveBeenCalledTimes(1);
       expect(userAgent).toContain('Node.js');
+      expect(userAgent).toContain('is_ci/false');
       expect(userAgent).toContain(process.version.slice(1));
       expect(userAgent).toContain(process.platform);
       expect(userAgent).toContain(process.arch);
-      expect(userAgent).toContain(`${pkg.name}/${pkg.version}`);
     });
 
     it('should return default user-agent for windows if nothing is set', () => {
       const isNode = jest.spyOn(env, 'isNode').mockReturnValue(false);
+      const isCI = jest.spyOn(env, 'isCI').mockReturnValue(false);
 
       // @ts-ignore
       const userAgent = HttpClient.getUserAgent({});
 
       expect(isNode).toHaveBeenCalledTimes(1);
+      expect(isCI).toHaveBeenCalledTimes(1);
       expect(userAgent).not.toContain('Node.js');
+      expect(userAgent).toContain('is_ci/false');
       expect(userAgent).not.toContain(process.version.slice(1));
-      expect(userAgent).toContain(`${pkg.name}/${pkg.version}`);
     });
 
     it('should set caller', () => {
       jest.spyOn(env, 'isNode').mockReturnValue(true);
+      jest.spyOn(env, 'isCI').mockReturnValue(false);
+      // @ts-ignore
+      jest.spyOn(fsScripts, 'getPaths').mockReturnValue(paths);
```

Comment:
it looks like you are running this line on every test - use `beforeEach` hook for it